main/lsp/LSPPreprocessor.h

```diff
@@ -22,6 +22,10 @@ struct QueueState {
     bool terminate = false;
     bool paused = false;
     int errorCode = 0;
+    // The latest edit version to be processed on the processing thread.
```

Comment:
`to be processed` wasn't clear for me.
Is that "scheduled to be processed" or "has been successully processed" or "currently being processed"?